tutorial/ndarrays.md

```diff
@@ -278,7 +278,7 @@ Some examples:
             a = array_create(2, (int64_t[]){2, 4}, nd_int64);
             int64_t array_dummy_0001[] = {1, 2, 3, 4, 5, 6, 7, 8};
             memcpy(a.nd_int64, array_dummy_0001, a.buffer_size);
-            b = array_slicing(a, 1, new_slice(1, 2, 1), new_slice(0, a.shape[1], 1));
+            b = array_slicing(a, 1, 2, new_slice(1, 2, 1), new_slice(0, a.shape[1], 1));
```

Comment:
@nhamidn  How do we differentiate between `a[:,0:1,:,2]` and `a[:,0,:,1:2]`

Reply:
@EmilyBourne Can we close this conversation?